intro-to-R-tidyverse/01-intro_to_r_new.Rmd

```diff
@@ -0,0 +1,255 @@
+---
+title: "Intro to R"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2020**
+
+## Objective for this notebook analysis: 
+
+We've [performed differential expression analyses](./scripts/00-setup-intro-to-R.R) on a pre-processed [astrocytoma microarray dataset](https://www.refine.bio/experiments/GSE44971/gene-expression-data-from-pilocytic-astrocytoma-tumour-samples-and-normal-cerebellum-controls).
+We performed three sets of contrasts:  
+
+1) `sex` category contrasting: Male vs Female  
+2) `tissue` category contrasting : Pilocytic astrocytoma tumor samples vs normal cerebellum samples  
+3) An interaction of both `sex` and `tissue`. 
+
+*Resources for learning R* 
+
+  - [Swirl, an interactive tutorial](https://swirlstats.com/)  
+  - [R Markdown](http://rmarkdown.rstudio.com)
+  - [R for Data Science](https://r4ds.had.co.nz/)  
+  - [Tutorial on R, RStudio and R Markdown](https://ismayc.github.io/rbasics-book/)  
+  - [Handy R cheatsheets](https://www.rstudio.com/resources/cheatsheets/)  
+  - [R Notebooks tutorial](https://bookdown.org/yihui/rmarkdown/)  
+
+## Set Up
+
+We saved these results to a tab separated values (TSV) file called `gene_results_GSE44971.tsv`.
+Its been saved to the `data` folder.  
+File paths are relative to where this notebook file (.Rmd) is saved.
+We saved our results file to a folder called `data`. 
+So we can reference it later, let's make a object in our environment with our 
+data directory name. 
+
+```{r}
+data_dir <- "data"
+```
+
+Magrittr pipe
+
+```{r}
+`%>%` <- dplyr::`%>%`
+```
+
+Let's declare our output folder name as its own object. 
+
+```{r}
+plots_dir <- "plots"
+```
+
+We can also create a directory if it doesn't already exist. 
+
+```{r}
+if(!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+```
+
+## Read in the differential expression analysis results file
+
+Here we are using a function `read_tsv` from a library of functions called `readr`. 
+
+```{r}
+stats_df <- readr::read_tsv(file.path(data_dir, 
+                                      "gene_results_GSE44971.tsv"))
+```
+
+Let's take a look at what this data.frame looks like: 
+
+```{r}
+stats_df
+```
+
+We can take a look at a column individually by using a `$`. 
+
+```{r}
+stats_df$contrast
+```
+
+If we want to see less values at once, we can use brackets with the indices of the values we'd like returned.
+
+```{r}
+stats_df$avg_expression[1:10]
+```
+
+Let's checkout how many rows of data there are per each contrast.
+
+```{r}
+summary(as.factor(stats_df$contrast))
+```
+
+## Tidyverse brief intro 
+
+Make a new variable of -log10 adjusted p values. 
```

Comment:
I'd just use p_values (not adjusted) for this plot; I don't want to get distracted by adjusted yet